br/backup-and-restore-overview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: BR Overview
+summary: Learn about the definition and functions of BR.
+aliases: ['/docs/dev/br/backup-and-restore-tool/','/docs/dev/reference/tools/br/br/','/docs/dev/how-to/maintain/backup-and-restore/br/','/tidb/dev/backup-and-restore-tool']
+---
+
+# BR Overview
+
+[BR](https://github.com/pingcap/br) (Backup & Restore) is a command-line tool for **distributed backup and restoration** of the TiDB cluster data. In addition to regular backup and restoration, you can also use BR for large-scale data migration as long as compatibility is ensured.
+
+This document describes BR's architecture, functions, and usage tips.
+
+## BR architecture
+
+BR sends a backup or restoration command to each TiKV node. After receiving the command, TiKV performs the corresponding backup or restoration operation.
+
+Each TiKV node needs to access the storage path during backup and restoration.
+
+![br-arch](/media/br-arch.png)
+
+For detailed information about the BR design, see [BR Design Principles](/br/backup-and-restore-design.md).
+
+## BR features
+
+This section describes BR features and performance indicators.
+
+### Back up TiDB cluster data
+
+This section describes backup features, impact of backup, and backup file types.
+
+#### Backup features
+
+- **Back up cluster snapshots**: A snapshot of a TiDB cluster contains transactionally consistent data at a specific time. You can back up snapshot data of a TiDB cluster using BR. For details, see [Back up TiDB cluster snapshots](/br/br-usage-backup.md#back-up-tidb-cluster-snapshots).
+- **Back up incremental data**: Incremental data of a TiDB cluster is differentiated data between the latest snapshot and the previous snapshot. Incremental data is smaller compared with full data and it is a supplementary to snapshot backup, which reduces the volume of backup data. For details, see [Back up incremental data](/br/br-usage-backup.md#back-up-incremental-data).
+- **Back up a database or a table**: In addition to snapshot and incremental data backup, BR supports backing up a database or table and filtering out unnecessary data. For details, see [Back up a database or table](/br/br-usage-backup.md#back-up-a-database-or-a-table).
+- **Encrypt backup data**: BR supports backup data encryption and amazon S3 server-side encryption. You can select an encryption method as required. For details, see [Encrypt backup data](/br/br-usage-backup.md#encrypt-backup-data).
```

Comment:
```suggestion
- **Encrypt backup data**: BR supports data encryption in the backup end or in the Amazon S3 storage server end. You can select an encryption method as needed. For details, see [Encrypt backup data](/br/br-usage-backup.md#encrypt-backup-data).
```

Reply:
missing

Reply:
Changed the Chinese in https://github.com/pingcap/docs-cn/pull/10099/files#diff-5f5519a7f44e95526e3253e724fdc87ff3e57461ceae2bfd6317dc96775fe3fcR30

Reply:
Comment updated

Reply:
server-side encryption seems like a fixed term. https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html